cpplint.py

```diff
@@ -6781,6 +6789,11 @@ def ParseArguments(args):
       recursive = True
     elif opt == '--includeorder':
       ProcessIncludeOrderOption(val)
+    elif opt == '--config':
+      global _config_filename
+      _config_filename = val
+      if os.path.basename(_config_filename) != _config_filename:
+        PrintUsage('Config file name must not include directory components.')
```

Comment:
IMO this isnt a good restriction, but I will merge as-is and @ACSimon33 maybe you want to remove it and possibly add some tests about this feature in a followup PR?

Reply:
If I understand this correctly, then removing this check would fix #208, right?